mergify_engine/mergify_pull.py

```diff
@@ -415,3 +415,7 @@ def merged(self):
     @property
     def merged_by(self):
         return self.g_pull.merged_by
+
+    @property
```

Comment:
This is not needed `get_existing_duplicate_pull` returns a `pyGithub.PullRequest`, not a `MergifyPull`

Reply:
Removed!